bot/cogs/doc.py

```diff
@@ -261,7 +261,7 @@ def __init__(self, bot: commands.Bot):
     @commands.group(name='docs', aliases=('doc', 'd'), invoke_without_command=True)
     async def docs_group(self, ctx: commands.Context, symbol: commands.clean_content = None) -> None:
         """Lookup documentation for Python symbols."""
-        await ctx.invoke(self.get_command)
+        await self.get_command(ctx, symbol)
```

Comment:
Commands can't be invoked directly, this needs to be `ctx.invoke`

```suggestion
        await ctx.invoke(self.get_command, symbol)
```